.rubocop.yml

```diff
@@ -149,7 +149,7 @@ Style/IfUnlessModifier:
   Description: Favor modifier if/unless usage when you have a single-line body.
   StyleGuide: https://github.com/bbatsov/ruby-style-guide#if-as-a-modifier
   Enabled: true
-  MaxLineLength: 100
+  MaxLineLength: 80
```

Comment:
I thought we decided the whole codebase should be rubocop clean.  Doesn't this mean this rule applies to all code, not just new code?